CONTRIBUTING.md

```diff
@@ -82,3 +81,26 @@ Before a pull request can be merged:
 * The feature branch must be rebased so it contains the latest changes from the target branch
 * The CI has to pass successfully
 * Every comment has to be addressed and resolved
+
+## Testing
+
+Tests are written using the *[Ginkgo](https://onsi.github.io/ginkgo/)* framework and here are some general advices when writing tests for the `release-service`:
+
+* Start the descriptions of `It` blocks in lowercase and try to be as more descriptive as possible.
```

Comment:
`s/as more descriptive/as descriptive/`

Reply:
every other line ends in `;` so maybe this one should too

Reply:
In addition, you can comment that `Context` blocks should start with `when`.

Reply:
Every line in this line did end with `;` so not sure what you mean David but it is changed now

Reply:
I meant other lists in the file, not Leandro's new addition.